CHANGELOG.md

```diff
@@ -1,60 +1,12 @@
 ## Unreleased
 
+## 0.32.1 (June 29, 2021)
+
 BUG FIXES:
 * ACLs: Support connecting to external Consul servers with an SNI header. [[GH-1005](https://github.com/hashicorp/consul-helm/pull/1005)]
 
 ## 0.32.0 (June 22, 2021)
 
-FEATURES:
```

Comment:
Reordering and putting breaking changes first.

Reply:
awesome, good call on this!